concepts/exceptions/.meta/config.json

```
@@ -0,0 +1,5 @@
+{
+  "blurb": "Exceptions should be thrown when an error that needs special handling occurs.",
```

Maybe make it slightly more matter-of-factly?

```suggestion
  "blurb": "Exceptions are thrown when an error that needs special handling occurs.",
```